jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/workitem/project/ProjectServiceTaskItemView.html

```diff
@@ -0,0 +1,28 @@
+<!--
+  ~ Copyright 2017 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<table>
+    <tbody>
+    <tr class="kie-tablelist__row" data-field="root">       
+        <td class="kie-tablelist__cell"><span data-field="icon"></span></td>
+        <td class="kie-tablelist__cell"><span data-field="name"></span></td>        
+        <td class="kie-tablelist__cell"><span data-field="additionalInfo"></span></td>
+        <td class="kie-tablelist__cell" width="15%">
```

Comment:
The td for the actions should use class _kie-tablelist__cell--actions_ and no width should be fine.